### ISSUE TITLE
No email validation in Midpoint create user process.

### ISSUE THREAD

**Issue by eea71** [OpenDataNode/open-data-node#160]
I created user in midpoint and put email 123 - is invalid. Was saved.

**Comment by peterklimo** [comment]
according to kvalalikova there is no email validator in the midpoint system. I cant remember to any usecase where given email is used

**Comment by peterklimo** [comment]
according to kvalalikova there is no email validator in the midpoint, but I do not think that email is used in any usecase

**Comment by peterklimo** [comment]
cording to discussion with kvalalikova there is no validation of parameters on the GUI layer. no validation of length, entered characters, no check for duplicity.
we need to decide if there is need to fix these issues, which will cost us additional effort

**Comment by eea71** [comment]
What does it mean "no validation of parameters on the GUI layer"? I think it is basic functionality to check if data are valid! 
![image](https://cloud.githubusercontent.com/assets/10993265/9107609/e294bff0-3c28-11e5-9bc7-a0368045c34b.png)
Picture above documents, that validations are here. Email address is one of REQUIRED data. It means probably important. Otherwise it doesn't have sense.
Minimally I think email address has to contain "@". This is minimal validation. But in this moment I can enter email address in shape "123". For future usage of email this must be.

**Comment by peterklimo** [comment]
yes only validation is if the parameter is mandatory or not, nothing else

On Thu, Aug 6, 2015 at 10:52 AM, Adriana Stanikova <notifications@github.com
